bin/src/cli.rs

```diff
@@ -180,13 +187,8 @@ pub enum SubCmd {
         #[clap(subcommand)]
         cmd: ListenerCmd,
     },
-    #[clap(name = "certificate", about = "certificate management")]
-    Certificate {
-        #[clap(subcommand)]
-        cmd: CertificateCmd,
-    },
-    #[clap(name = "query", about = "configuration state verification")]
-    Query {
+    #[clap(name = "certificates", about = "list, add and remove certificates")]
```

Comment:
Could you keep the `certificate` command instead of `certificates`

Reply:
yes